### PR TITLE
IDEX-3416: Add ability to set breakpoint marks in external classes

### PR DESCRIPTION
**Pull request by tolusha**

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ok

File: plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeExtension.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * http://www.eclipse.org/legal/epl-v10.html
  *
  * Contributors:
- *   Codenvy, S.A. - initial API and implementation
```
Comment (sleshchenko) [inline plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeExtension.java]: Please check license

Comment (tolusha) [inline plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeExtension.java]: Done

**Comment by vparfonov** [comment]
ok

**Pull request merged**